eng/python.gdnbaselines

```diff
@@ -1,5613 +1,5730 @@
 {
   "version": "1.0.0",
   "baselines": {
-    "baseline": {
-      "name": "baseline",
-      "createdDate": "2021-01-15 22:07:40Z",
-      "lastUpdatedDate": "2021-01-15 22:07:40Z"
+    "demobaseline": {
+      "name": "demobaseline",
+      "createdDate": "2021-01-29 00:29:11Z",
+      "lastUpdatedDate": "2021-01-29 00:29:11Z"
     }
   },
   "results": {
```

Comment:
Can we wait and update these baseline files once we get better suppression rules? I don't want these baseline files to constantly need updated based on any change and I'd like them to be much smaller if we get good placeholder values.

Reply:
We changed this because guardian team introduced some breaking changes which they do not recognize the errors in baseline file. 

The baseline file does not affect how we do the suppression. It is better to merge the baseline file first, and  come up with the suppression strategy in the meanwhile

Reply:
My expectation is that this baseline file will be much smaller once we do the suppressions. If we plan to update it then that is fine but I want to be careful that we don't have to have lots of folks updating this baseline file all the time.

Reply:
Currently, we have two purposes for putting baseline file here:
1. Leave time for owner to address the existing errors.
2. Catch new issue introduced to packages.

I think #2 is more crucial for our cred scan steps right now. The baseline file help us achieve the purpose, so it is necessary to check in now. 
As for #1, I plan to do some suppression on some obvious fake ones. Then we indeed need help from language owner.